app/src/test/java/com/example/tasky/feature_agenda/presentation/AgendaViewModelTest.kt

```diff
@@ -0,0 +1,83 @@
+package com.example.tasky.feature_agenda.presentation
+
+import com.example.tasky.common.domain.TaskyState
+import com.example.tasky.feature_login.domain.repository.UserRemoteRepository
+import io.mockk.MockKAnnotations
+import io.mockk.coEvery
+import io.mockk.impl.annotations.RelaxedMockK
+import junit.framework.Assert.assertEquals
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.TestScope
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import org.junit.Before
+import org.junit.Test
+
+class AgendaViewModelTest {
+    @RelaxedMockK
+    private lateinit var taskyState: TaskyState
+
+    @RelaxedMockK
+    private lateinit var userRemoteRepository: UserRemoteRepository
+
+    private lateinit var viewModel: AgendaViewModel
+
+    private val testDispatcher = StandardTestDispatcher()
+    private val testScope = TestScope(testDispatcher)
```

Comment:
All objects in a test should be initialized in setUp. You don't know if they contain internal state, but if they do, the state changes of one test could break another test if not reinitialized.